docs/source/user/talks.rst

```diff
@@ -0,0 +1,30 @@
+
+Talks and Tutorials
+===================
+
+Note: This is a selection of talks and tutorials have been given by members of
```

Comment:
Should this read `This is a selection of talks and tutorials THAT have been given by....`?